spec/observables/fromEvent-spec.ts

```diff
@@ -3,6 +3,7 @@ import { expectObservable } from '../helpers/marble-testing';
 import { fromEvent, NEVER, timer } from 'rxjs';
 import { mapTo, take, concat } from 'rxjs/operators';
 import { TestScheduler } from 'rxjs/testing';
+import sinon = require('sinon');
```

Comment:
you should be able to do a regular import these days with sinon FYI (`import * as sinon`)

Reply:
Ironically, the import wasn't even being used.